pro/cypress/e2e/step-definitions/searchCollectiveOffer.cy.ts

```diff
@@ -0,0 +1,13 @@
+import { Given, When } from '@badeball/cypress-cucumber-preprocessor'
+
+Given('I go to {string} view', (tab: string) => {
+  cy.findByText(tab).click()
+})
+
+When('I search for {string} in {string}', (option: string, filter: string) => {
```

Comment:
Ce ne serait pas "I select"  plutôt que "I go to" ?

Reply:
pour ce cas oui, c'est une step très orientée UI, ça serait logique de dire "I select"